Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/di/NetModule.java

```diff
@@ -19,28 +19,39 @@
 import com.android.volley.RequestQueue;
 import com.android.volley.toolbox.Volley;
 import com.github.adamantcheese.chan.BuildConfig;
-import com.github.adamantcheese.chan.core.cache.FileCache;
+import com.github.adamantcheese.chan.core.cache.CacheHandler;
+import com.github.adamantcheese.chan.core.cache.FileCacheV2;
+import com.github.adamantcheese.chan.core.cache.stream.WebmStreamingSource;
 import com.github.adamantcheese.chan.core.net.ProxiedHurlStack;
 import com.github.adamantcheese.chan.core.settings.ChanSettings;
 import com.github.adamantcheese.chan.core.site.http.HttpCallManager;
 import com.github.adamantcheese.chan.utils.Logger;
 import com.github.k1rakishou.fsaf.FileManager;
+import com.github.k1rakishou.fsaf.file.RawFile;
 
 import org.codejargon.feather.Provides;
 
 import java.io.File;
+import java.util.concurrent.TimeUnit;
 
+import javax.inject.Named;
 import javax.inject.Singleton;
 
 import okhttp3.OkHttpClient;
 
 import static com.github.adamantcheese.chan.utils.AndroidUtils.getAppContext;
 import static com.github.adamantcheese.chan.utils.AndroidUtils.getApplicationLabel;
 import static java.util.concurrent.TimeUnit.MINUTES;
-import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class NetModule {
     public static final String USER_AGENT = getApplicationLabel() + "/" + BuildConfig.VERSION_NAME;
+    public static final long DOWNLOADER_OKHTTP_TIMEOUT_SECONDS = 20L;
+    public static final long PROXIED_OKHTTP_TIMEOUT_SECONDS = 30L;
+    public static final long THREAD_SAVE_MANAGER_OKHTTP_TIMEOUT_SECONDS = 30L;
+    public static final String THREAD_SAVE_MANAGER_OKHTTP_CLIENT_NAME = "thread_save_manager_okhttp_client";
+    public static final String DOWNLOADER_OKHTTP_CLIENT_NAME = "downloader_okhttp_client";
+    private static final String FILE_CACHE_DIR = "filecache";
+    private static final String FILE_CHUNKS_CACHE_DIR = "file_chunks_cache";
```

Comment:
Done.

Reply:
Actually, do we really want to add it there. Because from what I can see in this commit 

>"Use a fileprovider to open videos in external apps"

I assume that the directory was added to that file so that an external video player is able to play video files in that directory. And since files in "file_chunks_cache" will never be played in a video player I guess I don't need to add it there?

Reply:
And Android already knows what to do with those files since they are in a cache directory. Android will just delete them when it's necessary.